app/views/cart/show.html.erb

```diff
@@ -30,9 +30,11 @@
       </td>
       <td>
         <center>
-          <%= button_to '+', "/cart?increment_decrement=increment&item_id=#{item.id}", method: :patch %>
```

Comment:
Thank you for catching this! I meant to make two, and then realized I don't need either... And I forgot to remove - You rock for catching it.